packages/realm/src/app-services/Credentials.ts

```diff
@@ -45,7 +46,25 @@ export class Credentials {
     this.internal = internal;
   }
 
+  /**
+   * Creates credentials for an anonymous user. These can only be used once - using them a second
+   * time will result in a different user being logged in. If you need to get a user that has already logged
+   * in with the Anonymous credentials, use {@linkcode App.currentUser} or {@linkcode App.allUsers}
+   * @return {Credentials} An instance of `Credentials` that can be used in {@linkcode App.logIn}.
```

Comment:
I suggest adding docs for the new parameter too.

```suggestion
   * @param reuse Reuse any existing anonymous user already logged in.
   * @return {Credentials} An instance of `Credentials` that can be used in {@linkcode App.logIn}.
```